rb/lib/selenium/webdriver/edge/legacy_support.rb

```diff
@@ -27,8 +27,7 @@ module LegacySupport
            send_keys_to_active_element get_window_handles get_current_window_handle
```

Comment:
Now that I've changed Remote calls to all use `Remote::Bridge` instead of `Remote::W3CBridge` we should get rid of this file entirely and move the methods directly into `Edge::Bridge`
Can you help with that as part of this PR, or should I do that? I'd like for this to be done together.